code/modules/projectiles/updated_projectiles/gun_system.dm

```diff
@@ -238,7 +238,7 @@
 	var/wdelay = wield_delay
 	//slower or faster wield delay depending on skill.
 	if(user.mind && user.mind.cm_skills)
-		if(user.mind.cm_skills.firearms == 0) //no training in any firearms
+		if(user.mind.cm_skills.firearms <= 0) //no training in any firearms
```

Comment:
Why this?

Reply:
It's consistent with other checks, and a failsafe because I think in some cases the firearms skill can be negative.